firestore/integration_test.go

```diff
@@ -1489,6 +1489,73 @@ func TestIntegration_CollectionGroupQueries(t *testing.T) {
 	}
 }
 
+func TestPartionQuery(t *testing.T) {
+	coll := collectionIDs.New()
+	ctx := context.Background()
+	h := testHelper{t}
+	client := integrationClient(t)
+	cr := client.Collection(coll)
+	subColl := coll + "-sub"
+	// Minimum partition size is 128.
+	documentCount := 128*2 + 127
+	var want []string
+	parents := []string{"abc-123", "def-456", "ghi-789"}
+	for i, parent := range parents {
+		dr1 := cr.Doc(parent)
+		scr := dr1.Collection(subColl)
```

Comment:
Why you're taking a doc and then collection from this doc? Doesn't look very correct.

Reply:
I'm saying you're initiating a doc `dr1` and then you're initiating a collection from this doc by `dr1.Collection`. Why not to initiate a connection right from `client`?

Reply:
Then we will have to use `Collection` function many times

Reply:
because this is subcollection for dr1 collection